src/amo/components/EditableCollectionAddon/index.js

```diff
@@ -107,30 +107,41 @@ export class EditableCollectionAddonBase extends React.Component<
           className,
         )}
       >
-        <img
-          className="EditableCollectionAddon-icon"
-          src={iconURL}
-          alt={addon.name}
-        />
-        <h2 className="EditableCollectionAddon-name">{addon.name}</h2>
-        <div className="EditableCollectionAddon-comments-icon">
-          <a
-            href="#editComment"
-            onClick={this.onEditNote}
-            className="EditableCollectionAddon-edit-note"
-          >
-            <Icon name="comments" />
-          </a>
+        <div className="EditableCollectionAddon-details">
+          <img
+            className="EditableCollectionAddon-icon"
+            src={iconURL}
+            alt={addon.name}
+          />
+          <h2 className="EditableCollectionAddon-name">{addon.name}</h2>
         </div>
-        <div className="EditableCollectionAddon-remove-button">
-          <Button
-            name={addon.id}
-            buttonType="alert"
-            onClick={this.onRemoveAddon}
-            micro
+        <div className="EditableCollectionAddon-buttons">
+          <div
+            className={makeClassName('EditableCollectionAddon-leaveNote', {
+              'EditableCollectionAddon-leaveNote--hidden': showNotes,
+            })}
           >
-            {i18n.gettext('Remove')}
-          </Button>
+            <Button
+              buttonType="action"
+              className="EditableCollectionAddon-leaveNote-button"
+              micro
+              name={`leaveNote-${addon.id}`}
+              onClick={this.onEditNote}
+            >
+              {i18n.gettext('Leave a note')}
+            </Button>
+          </div>
+          <div className="EditableCollectionAddon-remove">
```

Comment:
Do we need extra `div` around `Button`s for the css grid?

Reply:
You're right, we don't need this anymore. I think I did for a previous iteration, but not anymore. Thanks!